src/Avalonia.Visuals/Platform/IFontManagerImpl.cs

```diff
@@ -31,7 +31,7 @@ public interface IFontManagerImpl
         /// </returns>
         bool TryMatchCharacter(int codepoint, FontStyle fontStyle,
             FontWeight fontWeight,
-            FontFamily fontFamily, CultureInfo culture, out Typeface typeface);
+            FontFamily? fontFamily, CultureInfo? culture, out Typeface typeface);
```

Comment:
Can't typeface be null when this method returns false?

Reply:
Typeface is a struct and will have its default value.